ansible/roles/stack-sunbird/templates/ml-core-service.env

```diff
@@ -46,6 +46,15 @@ GOTENBERG_SERVICE_URL={{ml_gotenberg_url | default("http://gotenberg-service:300
 #  AWS_BUCKET_NAME=
 #  AWS_BUCKET_ENDPOINT=
 #  AWS_BUCKET_REGION=
+#
+## Oracle Cloud Configuration
+#
+#  CLOUD_STORAGE=OCI
+#  OCI_ACCESS_KEY_ID=
+#  OCI_SECRET_ACCESS_KEY=
+#  OCI_BUCKET_NAME=
+#  OCI_BUCKET_ENDPOINT=
+#  OCI_BUCKET_REGION=
 # To change the cloud config for azure, override below variable in common.yaml
 # For Example, 
 # ml_survey_cloud_config: |
```

Comment:
@VISHNUDAS-tunerlabs   understand its not part of the PR but where have we defined  **ml_survey_cloud_config** variable in private repo?